README.md

```diff
@@ -622,7 +622,7 @@ The maximum percentage of [minSize](#minSize) to unload during a given frame.
 ### .unloadPriorityCallback
 
 ```js
-unloadPriorityCallback = null : ( item ) => Number
+unloadPriorityCallback = null : ( itemA, itemB ) => Number
```

Comment:
It might if you think there's value in evicting based on similar/inverted rules to the download/parse priority? I hadn't looked into it yet, this was a mistake from editing the readme before noting that they were different implementations

Reply:
I created #192 to track the idea -- I figure maybe a small improvement to unloading tiles could be to sort first by tile depth and then by least recently used and then maybe by distance?